CZI.md

```diff
@@ -204,32 +170,38 @@ contributors.
 Quantitatively evaluating maintenance work can be tricky---some Issues
 or PRs take minutes to review while others can take days to weeks of
 effort---but we believe that there is value at looking at the total
-number of open Issues and PRs.  We will reduce this number by closing
-Issues and PRs faster than they are opened until a reasonable
+number of open Issues and PRs. 
+<!--we wrote this in our progress report--> 
+We will reduce this number by closing Issues and PRs faster than they are opened until a reasonable
 equilibrium is reached.  We will aim to hit the following metrics:
 
 - Initial response to all issues / new PRs within a week
 - Resolve majority of new issues / PRs within 1 month
 - ​​Reduce backlog of issues by 50 / quarter
 - ​​Reduce backlog of PRs by 50 / quarter
 
-During the current grant period we are developing a road map
+During the current grant period, we are developing a road map
 addressing how we will homogenize the API, implement "smart" composite
 Artists, and overhaul the data model.  In the second year we will
 begin to execute on that road map.  While we are still developing the
-road map, we expect it to include implementations the new
+road map, we expect it to include implementations of the new
 architecture, in both Matplotlib and domain specific libraries, ready
 for early users.
+<!-- this is not a coherent sentence--->
 
 Additionally, we anticipate that we will be able complete 5 projects
 of the scope of adding a color-blind preview within the scope of this
 grant.  We will identify the projects to be done based on our road map.
+<!-- you didn't really explain what that scope is -->
 
-The focus of the design work in current grant is on the internal API
+The focus of the design work in the current grant is on the internal API
 and data structures, in the second year we will begin to develop new
-general-use user facing APIs.
+general-use user facing APIs. <!--on top of implementing the old API under the hood?-->
 
 - design documentation about new user-facing API
+<!-- so the original plan had been to shim the API under the old user facing API - 
```

Comment:
I think we have to start shimming as validation that we have captured the full gamut of functionality and to hopefully reduce the amount of internal complexity. In addition once we are sure can reproduce the existing functionality, it is worth exploring if we can't come up with a public facing API informed by what we have learned.

Reply:
so the discussion we're currently negotiating is what levels of this API are public - like even the data sources are public API targeted at someone so I'm very unclear on what you even mean here by public that is different from the current pyplot/axes api. Like are we also going to carry our own ggplot like layer?

Reply:
I was being sloppy and meant "user facing" so what is described in the paragraph above.  I think some of my in-line notes and the text ended up merged.